apps/ledger-live-desktop/src/newArch/features/LedgerSyncEntryPoints/hooks/useEntryPoint.ts

```diff
@@ -61,6 +62,14 @@ export function useEntryPoint(entryPoint: EntryPoint, needEligibleDevice = true)
       },
       component: SettingsEntryPoint,
     },
+    [EntryPoint.postOnboarding]: {
+      enabled: featureLedgerSyncEntryPoints?.params?.postOnboarding ?? false,
+      page: AnalyticsPage.PostOnboarding,
+      onClick: () => {
+        track("banner_clicked", { banner: "Ledger Sync Activation", page: "PostOnboarding" });
```

Comment:
```suggestion
        track("banner_clicked", { banner: "Ledger Sync Activation", page: AnalyticsPage.PostOnboarding });
```

may work no? 

if this is the case can you update above EntryPoints also? plz